examples/release_highlights/plot_release_highlights_0_23_0.py

```diff
@@ -105,7 +105,7 @@
 X, y = make_blobs(random_state=rng)
 X = scipy.sparse.csr_matrix(X)
 X_train, X_test, _, y_test = train_test_split(X, y, random_state=rng)
-kmeans = KMeans(algorithm="elkan").fit(X_train)
+kmeans = KMeans(algorithm="elkan", n_init=10).fit(X_train)
```

Comment:
We can use the current default for `algorithm` and the future default for `n_init`.

```suggestion
kmeans = KMeans(n_init="auto").fit(X_train)
```

Reply:
Done! Thanks